meshnet/networker.go

```diff
@@ -21,7 +21,7 @@ type Networker interface {
 	UnSetMesh() error
 	// AllowIncoming creates an allowing fw rule for the given
 	// address
-	AllowIncoming(UniqueAddress) error
+	AllowIncoming(UniqueAddress, bool) error
```

Comment:
I would recommend to add description of what are expected parameters - added new parameter but it's meaning is unknown (because even name is not present)

Reply:
Added parameter names, should be more meaningful now.